thrusterCalibration/runThrusters.ino

```diff
@@ -1,20 +1,23 @@
 #include <Servo.h>
-
+//Intialize Servos
 Servo servo2;
 Servo servo3;
 Servo servo4;
 Servo servo5;
 Servo servo6;
 Servo servo7;
-
+//Group Servos int pairs
 Servo thrusterGroup1[] = { servo2, servo3 };
 Servo thrusterGroup2[] = { servo4, servo5 };
 Servo thrusterGroup3[] = { servo6, servo7 };
-
+//array of trusters for all the ones on the robot
```

Comment:
I just have a hard time reading/writing stuff like this.

A variable called `allThrusters` declared as an array.
If that wasn't enough, it has a list of all the servos.

Also 'thrusters'